### PR TITLE
[Documentation]: README.md edits

### PR DESCRIPTION
**Pull request by moutro**
In this Pull Request, I did the following:
 
- Restructure current README to remove props list and temp moved them to a separate docs/props.md directory
-  Make new docusaurus documentation site more prominent so users know to link to it
- Keep some code samples and custom calendar examples in this README

First draft for review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,207 +2,219 @@
...
-    'Septembre',
-    'Octobre',
-    'Novembre',
-    'Décembre'
-  ],
+  monthNames: ['Janvier', 'Février', 'Mars', 'Avril', 'Mai', 'Juin', 'Juillet', 'Août', 'Septembre', 'Octobre', 'Novembre', 'Décembre'],
   monthNamesShort: ['Janv.', 'Févr.', 'Mars', 'Avril', 'Mai', 'Juin', 'Juil.', 'Août', 'Sept.', 'Oct.', 'Nov.', 'Déc.'],
   dayNames: ['Dimanche', 'Lundi', 'Mardi', 'Mercredi', 'Jeudi', 'Vendredi', 'Samedi'],
   dayNamesShort: ['Dim.', 'Lun.', 'Mar.', 'Mer.', 'Jeu.', 'Ven.', 'Sam.'],
-  today: "Aujourd'hui"
```
Comment (Inbal-Tish) [inline README.md:87]: Why did you remove the "today"? It is necessary for the expandable calendar today button's label

File: README.md [hunk 2]
```diff
@@ -2,207 +2,219 @@
...
 ```
+## Customized Calendar Examples
 
-#### Date marking
+### Calendar
+<kbd>
+  <img src="https://github.com/wix-private/wix-react-native-calendar/blob/master/demo/assets/calendar.gif?raw=true">
+</kbd>
 
-**Disclaimer**: Make sure that `markedDates` param is immutable. If you change `markedDates` object content but the reference to it does not change calendar update will not be triggered.
```
Comment (Inbal-Tish) [inline README.md:158]: Is this note not important? Can we maybe add a info box with all the important notes along this file?

File: README.md [hunk 3]
```diff
@@ -2,207 +2,219 @@
...
-Multi-Dot marking
+### Multi-Dot marking
 
 <kbd>
  <img height=50 src="https://github.com/wix-private/wix-react-native-calendar/blob/master/demo/assets/marking4.png?raw=true">
 </kbd>
-<p></p>
-
-Use `markingType={'multi-dot'}` if you want to display more than one dot. Both the `<Calendar/>` and `<CalendarList/>` support multiple dots by using `dots` array in `markedDates` prop.
-The property `color` is mandatory while `key` and `selectedColor` are optional. If key is omitted then the array index is used as key. If `selectedColor` is omitted then `color` will be used for selected dates.
```
Comment (Inbal-Tish) [inline README.md:189]: This is also an important note, no? Or is there another place for it?

File: README.md [hunk 4]
```diff
@@ -211,400 +223,46 @@ Period marking
...
-  }}
-/>
-```
 
 <kbd>
   <img height=350 src="https://github.com/wix-private/wix-react-native-calendar/blob/master/demo/assets/multi-marking.png?raw=true">
 </kbd>
-<p></p>
 
-Keep in mind that different marking types are not compatible. You can use just one marking style for a calendar.
```
Comment (Inbal-Tish) [inline README.md:318]: Also important note

File: docs/props.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# API Reference
+
+## Props
+
+| **Name** | **Type** | **Default** | **Description** | **Code Example** |
+|----------|----------|----------|----------|----------|
+| `current`                 | `string` or `Date`  |          | The current date that is displayed in the calendar.         | `<Calendar current={new Date()}/>`         |
```
Comment (Inbal-Tish) [inline docs/props.md:7]: The type is 'string', in a specific format 'YYYY-MM-dd', Date is invalid

File: docs/props.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+|  `minDate`                | `string` or `Date` |          | The minimum date that can be selected in the calendar.         | `<Calendar minDate={new Date(2022, 5, 15)}/>`         |
+| `maxDate`                 | `string` or `Date` |          | The maximum date that can be selected in the calendar.         | `<Calendar maxDate={new Date(2022, 6, 15)}/>`         |
+| `firstDay`                | `integer`         | 0         | The first day of the week. 0 for Sunday, 1 for Monday, 2 for Tuesday, etc.         | `<Calendar firstDay={1}/>`         |
+| `markedDates`             | `object`       |          | Allows you to mark certain dates on the calendar and customize their appearance. The keys of the object should be the date strings (yyyy-mm-dd) and the values can be a boolean or an object.         |          |
```
Comment (Inbal-Tish) [inline docs/props.md:11]:
```suggestion
| `markedDates`             | `object`       |          | Allows you to mark certain dates on the calendar and customize their appearance. The keys of the object should be the date strings (yyyy-MM-dd) and the values can be a boolean or an object.         |          |
```

File: README.md (hunk 1 above)
Comment (moutro) [reply README.md:87]: Hi @Inbal-Tish - my apologies for the delays. Will work on these asap. I added it back.

File: README.md (hunk 3 above)
Comment (moutro) [reply README.md:189]: I moved these from the main repo readme into the docusaurus pages.

File: README.md (hunk 4 above)
Comment (moutro) [reply README.md:318]: moved to Calendar.md docusaurus docs

File: docs/props.md (hunk 5 above)
Comment (moutro) [reply docs/props.md:7]: Thinking we might want to remove this file now unless you like the API reference to be a separate section in the TOC of the docusaurus docs?

Comment (Inbal-Tish) [reply docs/props.md:7]: No. I prefer to remove it so we won't have to maintain another file. We have the .api.json files for anyone that is looking at the source code and these are used to generate the docs

File: README.md [hunk 7]
```diff
@@ -2,69 +2,147 @@
...
 
+## A declarative cross-platform React Native calendar component for iOS and Android.
+
 [![Version](https://img.shields.io/npm/v/react-native-calendars.svg)](https://www.npmjs.com/package/react-native-calendars)
 [![Build Status](https://travis-ci.org/wix/react-native-calendars.svg?branch=master)](https://travis-ci.org/wix/react-native-calendars)
 
-This module includes various customizable **React-Native** calendar components.
+<br>
+
+This module includes information on how to use this customizable **React-Native** calendar component.
```
Comment (Inbal-Tish) [inline README.md:12]:
```suggestion
This module includes information on how to use this customizable **React Native** calendar component.
```

File: README.md [hunk 8]
```diff
@@ -2,69 +2,147 @@
...
+
+
+> ### **Official documentation**  
+> This README provides basic examples of how to get started with `react-native-calendars`. For detailed information, refer to the [official documentation site](https://wix.github.io/react-native-calendars/docs/intro).
 
-The package is both **Android** and **iOS** compatible.
 
-**See our new [Docs site](https://wix.github.io/react-native-calendars/)**
+## Features ✨
+* Pure JS. No Native code required
```
Comment (Inbal-Tish) [inline README.md:24]: We should add the marking dates (dot, multi-dot, period, multi-period and custom) options

File: README.md [hunk 9]
```diff
@@ -2,69 +2,147 @@
...
 
-`import {`[Calendar](#calendar), [CalendarList](#calendarlist), [Agenda](#agenda)`} from 'react-native-calendars';`
+## Some Code Examples
 
-All parameters for components are optional. By default the month of current local date will be displayed.
+Here are a few code snippets that demonstrate how to use some of the key features of react-native-calendars:  
 
-Event handler callbacks are called with `calendar objects` like this:
 
+### Customize the appearance of the calendar:
```
Comment (Inbal-Tish) [inline README.md:96]: Feels like this should come after "Creating a basic calendar with default settings", no?

File: README.md [hunk 10]
```diff
@@ -81,128 +159,79 @@ LocaleConfig.locales['fr'] = {
...
+      }}
+    />
+  );
+};
 
-#### Basic parameters
+export default App;
+```
 
+### Displaying the calendar in a custom style:
```
Comment (Inbal-Tish) [inline README.md:189]: More like adding a global theme to the calendar (we have the custom style example above...)

File: README.md (hunk 7 above)
Comment (moutro) [reply README.md:12]: ✅  - Thanks for catching that. Will fix it.

File: README.md (hunk 8 above)
Comment (moutro) [reply README.md:24]: Good point! Will add them.

File: README.md (hunk 9 above)
Comment (moutro) [reply README.md:96]: Yes, definitely. It's the logical order for sure. Thanks!

File: docs/props.md (hunk 5 above)
Comment (moutro) [reply docs/props.md:7]: Yes, makes sense to remove it - doesn't serve any purpose.

**Comment by moutro** [comment]
@Inbal-Tish - Please let me know if there is anything currently on the repo readme that you want added to the docusaurus pages.

**Review by Inbal-Tish — APPROVED** [review]

**Comment by Inbal-Tish** [comment]
> @Inbal-Tish - Please let me know if there is anything currently on the repo readme that you want added to the docusaurus pages.

Looks really great! Thank you so much!
